### PR TITLE
terraform-plan: add page

### PR DESCRIPTION
**Pull request by psyashes**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known): 1.0.11**

### CODE REVIEW

File: pages/common/terraform-plan.md
Comment (marchersimon) [inline pages/common/terraform-plan.md:3]:
```suggestion
> Generate and show Terraform execution plans.
```

File: pages/common/terraform-plan.md
Comment (marchersimon) [inline pages/common/terraform-plan.md:6]:
```suggestion
- Generate and show the execution plan in the current directory:
```

File: pages/common/terraform-plan.md
Comment (marchersimon) [inline pages/common/terraform-plan.md:10]: I don't exactly understand what this does, but is this still correct?
```suggestion
- Show a plan to destroy all remote objects that currently exist:
```

File: pages/common/terraform-plan.md
Comment (marchersimon) [inline pages/common/terraform-plan.md:14]:
```suggestion
- Show a plan to update the Terraform state and output values:
```

File: pages/common/terraform-plan.md
Comment (marchersimon) [inline pages/common/terraform-plan.md:20]: The `{{}}` will hightlight arguments. Also, can you specify mutliple pairs like this?
```suggestion
`terraform plan -var '{{name1}}={{value1}}' '{{name2}}={{value2}}'`
```

File: pages/common/terraform-plan.md
Comment (marchersimon) [inline pages/common/terraform-plan.md:24]:
```suggestion
`terraform plan -target {{resource_type.resource_name[instance index]}}`
```

File: pages/common/terraform-plan.md
Comment (marchersimon) [inline pages/common/terraform-plan.md:26]:
```suggestion
- Output a plan as JSON:
```

File: pages/common/terraform-plan.md
Comment (marchersimon) [inline pages/common/terraform-plan.md:30]:
```suggestion
- Write a plan to a specific file:
```

File: pages/common/terraform-plan.md
Comment (marchersimon) [inline pages/common/terraform-plan.md:32]:
```suggestion
`terraform plan -no-color > {{path/to/file}}`
```
`>>` appends to a file, `>` overwrites it. I assume you meant to overwrite it.

File: pages/common/terraform-plan.md
Comment (psyashes) [reply pages/common/terraform-plan.md:10]: Yes, it's almost right.

File: pages/common/terraform-plan.md
Comment (psyashes) [reply pages/common/terraform-plan.md:20]: Thank you for reviewing.
`-var` accepts only one argument. If you want to pass several arguments, do like this.

```bash
terraform plan -var '{{name1=value1}}' -var '{{name2=value2}}'
```

I'll fix it as above.

File: pages/common/terraform-plan.md
Comment (marchersimon) [inline pages/common/terraform-plan.md:20]:
```suggestion
`terraform plan -var '{{name1}}={{value1}}' -var '{{name2}}={{value2}}'`
```
In this case we don't put the `=` in token syntax, because it is nothing the user can change

**Review by marchersimon — APPROVED** [review]
Just one last suggestion
Thanks @psyashes

File: pages/common/terraform-plan.md
Comment (psyashes) [reply pages/common/terraform-plan.md]: Thanks @marchersimon 
I'll fix it and wait for another approval👍

File: pages/common/terraform-plan.md
Comment (navarroaxel) [inline pages/common/terraform-plan.md:6]:
```suggestion
- Generate and show the execution plan in the currently directory:
```

File: pages/common/terraform-plan.md
Comment (navarroaxel) [inline pages/common/terraform-plan.md:32]:
```suggestion
`terraform plan -no-color > {{path/to/file}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**
